docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -47,6 +47,19 @@ In virtual environments, high CPU steal indicates that the virtual machines migh
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/buckets-memory-and-storage/storage.html[Storage]
 
+[#CB90044]
+=== Babysitter Managed Process Crash (CB90043)
+
+*Background*: Babysitter is part of Couchbase Server's cluster manager which is responsible for maintaining a variety of Couchbase Server processes.
+If any of the processes managed by the babysitter die, it is responsible for restarting them.
+
+*Condition*: A process managed by babysitter crashes.
+
+*Remediation*: A process can crash for a number of reasons, so if it happens once it is not indicative of a Couchbase Server issue.
```

Comment:
I'd perhaps make this a little more explicit - when I was scanning it I read it as "if it happens it's not a Couchbase Server issue". Perhaps "if it happens once or twice it may not be indicative..."